front/app/components/SeatInfo/TrackerInfo.tsx

```diff
@@ -0,0 +1,155 @@
+import React from 'react';
+import { rgba } from 'polished';
+
+// Components
+import {
+  Box,
+  Icon,
+  Text,
+  IconTooltip,
+  colors,
+} from '@citizenlab/cl2-component-library';
+
+// Hooks
+import useAppConfiguration from 'api/app_configuration/useAppConfiguration';
+import useSeats from 'api/seats/useSeats';
+
+// Types
+import { SeatInfoProps, SeatNumbersType, SeatTypeMessageDescriptor } from '.';
+
+// Intl
+import { FormattedMessage, useIntl } from 'utils/cl-intl';
+import messages from './messages';
+
+// Utils
+import { isNil } from 'utils/helperUtils';
+
+const TrackerInfo = ({ seatType }: SeatInfoProps) => {
+  const { formatMessage } = useIntl();
+  const { data: appConfiguration } = useAppConfiguration();
+  const { data: seats } = useSeats();
+
+  const maximumSeatNumbers: SeatNumbersType = {
+    admin:
+      appConfiguration?.data.attributes.settings.core.maximum_admins_number,
+    collaborator:
+      appConfiguration?.data.attributes.settings.core.maximum_moderators_number,
+  };
+  const maximumSeatNumber = maximumSeatNumbers[seatType];
+
+  // Maximum seat number being null means that there are unlimited seats so we don't show the seat info
+  if (isNil(maximumSeatNumber) || !seats) {
+    return null;
+  }
+
+  let currentSeatNumber = {
+    admin: seats.data.attributes.admins_number,
+    collaborator: seats.data.attributes.project_moderators_number,
+  }[seatType];
+  const additionalSeats = currentSeatNumber - maximumSeatNumber;
+  const showAdditionalSeats = additionalSeats > 0;
+
+  const seatTypeMessage: SeatTypeMessageDescriptor = {
+    admin: messages.currentAdminSeatsTitle,
+    collaborator: messages.currentCollaboratorSeatsTitle,
+  };
+  const currentSeatTypeTitle = seatTypeMessage[seatType];
+  const tooltipMessages: SeatTypeMessageDescriptor = {
+    admin: messages.includedAdminToolTip,
+    collaborator: messages.includedCollaboratorToolTip,
+  };
+  const tooltipMessage = tooltipMessages[seatType];
+
+  const seatTypeInfoMessage = {
+    admin: messages.adminInfoTextWithoutBilling,
+    collaborator: messages.collaboratorInfoTextWithoutBilling,
+  }[seatType];
+
+  // Show maximum number of seats if user has used more for this value
+  if (currentSeatNumber >= maximumSeatNumber) {
+    currentSeatNumber = maximumSeatNumber;
+  }
```

Comment:
Yeah, there is some confusing code in the feature generally because we are handling two iterations but should be fine once we are stable with the second